schema/resources/assertion.json

```diff
@@ -0,0 +1,19 @@
+{
+    "title": "arxiv-external-links",
```

Comment:
Since each schema is for a particular type of resource, will want to give this a more specific name (e.g. ``RelationAssertion``?)